pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -980,6 +981,9 @@ func (m *kubeGenericRuntimeManager) computePodActions(ctx context.Context, pod *
 						reason: reasonUnknown,
 					}
 				}
+			} else if containerStatus.State == kubecontainer.ContainerStateCreated {
```

Comment:
Maybe surround the change with `pleg.IsEventedPLEGInUse()`? https://github.com/kubernetes/kubernetes/pull/122778/files#r1459564818

Reply:
let me update it

Reply:
Updated.